internal/command/command.go

```diff
@@ -22,6 +22,69 @@ type Command interface {
 	Run() error                // Run runs the command
 }
 
+type PlatformBuilder interface {
```

Comment:
It is a good question. If we want to allow outside tool to directly use fyne-cross code, yes, otherwise, no. We can keep them private at the moment, and when we move things to fyne we can decide on a struct by struct basis.

Reply:
I have unexported all the new type and functions. I have kept the exported type for `Command` as that should be addressed when migrating to cli.App.